### ISSUE TITLE
cosmos create_database unable to get pipeline in cosmos e2e tests

### ISSUE THREAD

**Issue by ctaggart** [Azure/azure-sdk-for-rust#273]
This `unwrap()` is panicking when running `cargo test --features test_e2e` for cosmos`.

```
     Running tests\attachment_00.rs (C:\Users\cataggar\io\azure-sdk-for-rust\target\debug\deps\attachment_00-d3a656dafad6c073.exe)     

running 1 test
test attachment ... FAILED

failures:

---- attachment stdout ----
thread 'attachment' panicked at 'called `Option::unwrap()` on a `None` value', C:\Users\cataggar\io\azure-sdk-for-rust\sdk\cosmos\src\clients\cosmos_client.rs:158:14
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    attachment

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

error: test failed, to rerun pass '--test attachment_00'
```

https://github.com/Azure/azure-sdk-for-rust/blob/8784ea58e16261a80b47cb525edb9808052813a8/sdk/cosmos/src/clients/cosmos_client.rs#L146-L171

**Comment by ctaggart** [comment]
This actually fails multiple tests:
```
cargo test --features test_e2e --no-fail-fast
```

**Comment by ctaggart** [comment]
It is fixed by #124.
